docs/tutorials/unraid.md

```diff
@@ -90,3 +90,19 @@ If you would like to schedule a periodic scan of your library, use the [`searchC
 If you would like to set up RSS scans, use the [`rssCadence`](../basics/options.md#rsscadence) option.
 
 If you would like to automatically check for cross-seeds when a download finishes, learn how to [set up with automatic searches](../basics/daemon#set-up-automatic-searches-for-finished-downloads).
+
+## Auto-Restart
+
+`cross-seed` by default will stop the container if a configuration option is suspected to be bad. This includes bad config options for the torrent client.
```

Comment:
Adding this blurb is all that makes sense.

Reply:
If a user reads this, then their next question is going to be "why does that matter to me", and if they know why that matters to them, then it'll be "how do I prevent those problems from happening". I think the docs added address those questions in a user-friendly way.